curvefs/src/metaserver/copyset/meta_operator.cpp

```diff
@@ -224,7 +230,7 @@ void GetOrModifyS3ChunkInfoOperator::OnApply(int64_t index,
         TYPE##Response response;                                             \
         auto status = node_->GetMetaStore()->TYPE(                           \
             static_cast<const TYPE##Request*>(request_), &response);         \
-        node_->GetMetric()->OnOperatorComplete(                              \
+        node_->GetMetric()->OnOperatorCompleteFromLog(                       \
```

Comment:
you forget `GetOrModifyS3ChunkInfoOperator::OnApplyFromLog`

Reply:
> you forget `GetOrModifyS3ChunkInfoOperator::OnApplyFromLog`

fixed